01-messaging.md

```diff
@@ -13,6 +13,7 @@ All data fields are unsigned big-endian unless otherwise specified.
 
   * [Connection Handling and Multiplexing](#connection-handling-and-multiplexing)
   * [Lightning Message Format](#lightning-message-format)
+  * [Wire Type-Length-Value Format](#wire-type-length-value-format)
```

Comment:
nit: maybe no need to specify `wire` now that we agree to use this everywhere. Simply `Type-Length-Value Format`?